README.md

```diff
@@ -23,7 +23,7 @@ cd contract && npm install
 cd ../client && npm install
 cd ../server && npm install
 ```
-
+The subsequent steps require [docker compose](https://docs.docker.com/compose/install/linux/) and [NodeJS >= 16](https://nodejs.org/en/blog/release/v16.16.0/)
```

Comment:
why link to the linux installation and the blog article of a specific node release?

```suggestion
The subsequent steps require [docker compose](https://docs.docker.com/compose/install) and [NodeJS >= 16](https://nodejs.org/en/download)
```

Reply:
thought I saw other OSes on that docker site, too. The release announcement has all the installers of the mentioned version, we can link to the latest LTS also, I don't care.